docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
```

Comment:
(opening thread here randomly)

I think we should also add a page on the writing policies section. It would state what the audit scanner is and pointing to the explanation, and documenting `spec.backgroundAudit`, saying that is enabled by default, and that one should disable if the policy hits the intrinsic limitations documented in the explanation page.